### PR TITLE
Enhancement/tokens table

### PR DESCRIPTION
**Pull request by olgakup**
This PR implements new address tokens table design . This pr will address issue #530

### CODE REVIEW

**Comment by olgakup** [comment]
@carolinaf86 , this is work in progress. I still need to finish the rest of the mobile ui and test it. Can you please take a look at the first commit. I added method to token-page extension to return BN type for total tokens. Just want to make sure I didn't break anything. Thank YOU!

File: apps/explorer/src/core/api/apollo/extensions/token-page.ext.ts [hunk 1]
```diff
@@ -18,14 +19,38 @@ export class TokenPageExt_items implements TokenPage_items {
...
+    this.priceChange24h = proto.priceChange24h
   }
 
   get currentPriceBN(): BN | null {
     return this.currentPrice ? new BN(this.currentPrice) : null
   }
 
-  get balanceBN(): BN | null {
-    return this.balance ? new BN(this.balance) : null
+  get formattedBalance(): string {
```
Comment (carolinaf86) [inline apps/explorer/src/core/api/apollo/extensions/token-page.ext.ts]: @olgakup this is where we can add BigNumber conversion and other calculations that are likely to be used in multiple places

File: apps/explorer/src/modules/addresses/components/TableAddressTokensRow.vue [hunk 2]
```diff
@@ -52,31 +64,58 @@ export default class TableAddressTokensRow extends Mixins(StringConcatMixin) {
...
+
+  get currPrice(): string {
+    return this.token.currentPriceBN ? this.getRoundNumber(this.token.currentPriceBN) : '0.00'
+  }
+
+  get  usdValue(): string {
+    return this.getRoundNumber(this.token.usdValueBN)
+  }
+
+  get changeInPrice(): string {
```
Comment (carolinaf86) [inline apps/explorer/src/modules/addresses/components/TableAddressTokensRow.vue]: This seemed specific to the component so I didn't move it to the type class. But if in future we need to use it in multiple components we could move it to TokenPageExt_items class instead.

File: apps/explorer/src/core/api/apollo/extensions/token-page.ext.ts (hunk 1 above)
Comment (carolinaf86) [reply apps/explorer/src/core/api/apollo/extensions/token-page.ext.ts]: Sorry @olgakup, now I see this is exactly what you did already with the page class, so no need for me to explain!

File: apps/explorer/src/modules/addresses/components/TableAddressTokensRow.vue (hunk 2 above)
Comment (olgakup) [reply apps/explorer/src/modules/addresses/components/TableAddressTokensRow.vue]: This actually can be good to add it tot he ext_item. This is used on tokens' page, and in token detail page.

**Comment by olgakup** [comment]
@carolinaf86 i finished mobile ui, can you please add logo to the token extension? 
Also i discovered that, Mixin in token row is not working for value formatting, we should fix it before merging too.

**Comment by carolinaf86** [comment]
@olgakup, I've added in logos that we store in eth_list_contract_metadata table to the TokenPageExt_items. Right now, the test dataset doesn't have any date in that table, so rows need to be manually added to test. If you manually insert rows to token_exchange_rates too all the functionality of TableAddressTokens can be demonstrated.

**Comment by olgakup** [comment]
@carolinaf86 thank you. I will take a look!

**Comment by olgakup** [comment]
Hey I am trying to figure out whats going on. But once i merged with develop i can't load ethvm.lan  Even though all logs show that all images running fine. I also pruned old images and rebuild docker multiple times, and no help.

**Comment by olgakup** [comment]
@carolinaf86 i saw that you added  priceChange24h , symbol and class to the ext. These values are also used in TokenTableRow.vue  ( tokens page). If you want we can add them in this pr too, otherwise I am just going to add loading component for the address tokens and we can merge.

**Comment by olgakup** [comment]
@carolinaf86, the more and more I am thinking about adding priceChange24h to the tokens page, the more I realize that we need bigger main-net dataset to test the tokens exchange rate part. So it is up to you. This should be ready for merge.

**Comment by carolinaf86** [comment]
@olgakup maybe it would be best to go ahead and merge so that we can test this in main-net staging when ready? I'll add the same getters to the TokenExchangeRatePageExt_items for TokenTableRow.vue. We can't reuse here as the views use different models (Token vs TokenExchangeRate. I'm gonna make a task to rename "Token" to "TokenBalance" post-beta as this is more accurate.) I'll create a TokenUtils class to avoid duplicating the code.

**Pull request merged**